test.sh

```diff
@@ -88,9 +88,13 @@ for version in "${meteor_versions[@]}"; do
 	elif [[ "${version}" == 2.3.3 ]] || [[ "${version}" == 2.3.4 ]]; then
 		node_version='14.17.4'
 
-	# Versions >= 2.3.5 need Node 14.17.5
-	else
+	# Versions 2.3.5 need Node 14.17.5
```

Comment:
```suggestion
	# Version 2.3.5 needs Node 14.17.5
```

Reply:
Done. I also updated other lines with the same typo.